website_menu_by_user_status/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Website Menu By User Status
+===========================
+
+The module manages display website menu entries, depending if the user is
+logged or not.
+The selection of the display status can be chosen logged and/or not.
+
+Installation
+============
+
+To install this module, just click to install butto
+
+Configuration
+=============
+
+#. you must activate the developer mode
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Website > Configuration > Settings > Pages.
+#. Select the page for which you would like to hide the menu.
+#. In the `Related Menu Items` table, check whether the menu item is visible for logged/unlogged users.
+
+#. remove default filter to edit website menu line
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/website/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+
+Contributors
+------------
+* Bruno Joliveau <bruno.joliveau@savoirfairelinux.com>
+* Jordi Riera <jordi.riera@savoirfairelinux.com>
+* Meyomesse Gilles <meyomesse.gilles@gmail.com>
+* David Dufresne
+
+More information
```

Comment:
update README according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst